libse/SubtitleFormats/DigiBeta.cs

```diff
@@ -21,51 +22,49 @@ public override string ToText(Subtitle subtitle, string title)
             var sb = new StringBuilder();
             foreach (Paragraph p in subtitle.Paragraphs)
             {
-                sb.AppendLine(string.Format(paragraphWriteFormat, EncodeTimeCode(p.StartTime), EncodeTimeCode(p.EndTime), p.Text.Replace(Environment.NewLine, "\t")));
+                sb.AppendLine(string.Format(paragraphWriteFormat, EncodeTimeCode(p.StartTime), EncodeTimeCode(p.EndTime), p.Text.Replace(Environment.NewLine, NewLineToken)));
             }
             return sb.ToString().Trim();
         }
 
         public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string fileName)
         {
             _errorCount = 0;
-
             subtitle.Paragraphs.Clear();
+            char[] splitChar = { ' ' };
+            int lineNumber = 1;
+
             foreach (string line in lines)
             {
-                if (RegexTimeCode.IsMatch(line) && line.Length > 24)
+                if (line.Length > 24 && RegexTimeCode.IsMatch(line))
                 {
-                    string[] parts = line.Substring(0, 11).Split(new[] { ' ' }, StringSplitOptions.RemoveEmptyEntries);
-                    if (parts.Length == 4)
+                    try
                     {
-                        try
-                        {
-                            var start = DecodeTimeCodeFramesFourParts(parts);
-                            parts = line.Substring(12, 11).Split(new[] { ' ' }, StringSplitOptions.RemoveEmptyEntries);
-                            var end = DecodeTimeCodeFramesFourParts(parts);
-                            var paragraph = new Paragraph
-                            {
-                                StartTime = start,
-                                EndTime = end,
-                                Text = line.Substring(24).Trim().Replace("\t", Environment.NewLine)
-                            };
+                        string[] startTokens = line.Substring(0, 11).Split(splitChar, StringSplitOptions.RemoveEmptyEntries);
+                        string[] endTokens = line.Substring(12, 11).Split(splitChar, StringSplitOptions.RemoveEmptyEntries);
 
-                            subtitle.Paragraphs.Add(paragraph);
-                        }
-                        catch
+                        subtitle.Paragraphs.Add(new Paragraph
                         {
-                            _errorCount++;
-                        }
+                            Number = lineNumber++,

```

Comment:
Initialize number to avoid unnecessary call to subtitle.Renumber() (even when parsing fails)